src/Native/System.Net.Http.Native/pal_curlinit.h

```diff
@@ -15,3 +15,4 @@
  * Returns 0 on success and non-zero on failure.
  */
 extern "C" int32_t EnsureCurlIsInitialized();
+extern "C" int64_t GetCountIdentityCertsWithLabel(const char *label);
```

Comment:
I think  I screwed up while doing rebase. This was removed sometime back. Removed it